Author: Modestas Vainius <modax@debian.org>
Description: do not ship marble plugins with useless R(UN)PATH=/usr
Bug-Debian: http://bugs.debian.org/cgi-bin/bugreport.cgi?bug=582139
Forwarded: no
Last-Update: 2010-06-07

--- a/marble/MarbleMacros.cmake
+++ b/marble/MarbleMacros.cmake
@@ -36,7 +36,7 @@ set_target_properties( ${_target_name} P
                        INSTALL_RPATH_USE_LINK_PATH TRUE  
                        SKIP_BUILD_RPATH TRUE 
                        BUILD_WITH_INSTALL_RPATH TRUE 
-                       INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/${LIB_SUFFIX} )
+                     )
 
 endmacro( marble_add_plugin _target_name )
 
@@ -81,7 +81,7 @@ set_target_properties( ${_target_name} P
                        INSTALL_RPATH_USE_LINK_PATH TRUE  
                        SKIP_BUILD_RPATH TRUE 
                        BUILD_WITH_INSTALL_RPATH TRUE 
-                       INSTALL_RPATH ${CMAKE_INSTALL_PREFIX}/${LIB_SUFFIX} )
+                     )
 
 endmacro( marble_add_designer_plugin _target_name )
 
